Rakefile

```diff
@@ -65,8 +69,7 @@ task :generate_policy_list do
       puts "Skipping #{pp.parsed_name} because publish flag set to a value other than 'true'"
     else
       # Get datetime for last time file was modified
-      commits = github_client.commits(repo_name, branch, path: file)
-      updated_at = commits.first.commit.author.date.utc.iso8601 if !commits.empty?
+      updated_at = `git log -1 --format="%cI" -- #{file}`.strip
```

Comment:
What's the reason for changing to a bash command instead of using the built-in github libraries?

Reply:
My bad. I changed it for local testing. revert it: https://github.com/flexera-public/policy_templates/pull/2813/commits/7c7ccccea6e8486660a70d7b13622d116a84cbf4